### PR TITLE
Added 'RedHerd Framework'

### PR DESCRIPTION
**Pull request by Peco602**
RedHerd is a collaborative and serverless framework for orchestrating a geographically distributed group of assets capable of simulating complex offensive cyberspace operations.

### CODE REVIEW

**Review by mashb1t — APPROVED** [review]
Hey @Peco602,
it took a little time to get a glimpse of what the purpose of this toolset/framework is but when digging deeper into the paper it became clear. Especially using a drone as asset is an interesting idea.
LGTM but i'd like to have a second opinion from another maintainer.

**Comment by Peco602** [comment]
Thank you very much! Your words are really appreciated. We have also presented the RedHerd Framework at BlackHat Europe 2021 Arsenal. You can find the video of the presentation on YouTube.

**Review by veggiemonk — APPROVED** [review]
Thank you! Nice project 👍

**Pull request merged**
